src/components/views/Login.tsx

```diff
@@ -164,7 +164,7 @@ export default withRouter(
                         onClick={() => this.performOAuth(OAuthProvider.TGForums)}
                         key={OAuthProvider.TGForums}>
                         <img
-                            src="https://tgstation13.org/favicon.ico"
+                            src="https://cdn.discordapp.com/attachments/484170915253321734/785966334952341584/tglogo-white.svg"
```

Comment:
I'd rather not pull externals dependencies, if discord goes down, it shouldn't be affecting the webpanel, svg-loader should be working and you should be able to

import tgicon from "../../../images/tglogo.svg";

<svg src={tgicon}>